src/Http/Http.Extensions/gen/RequestDelegateGeneratorSources.cs

```diff
@@ -112,21 +114,8 @@ private static async ValueTask<(bool, T?)> TryResolveBodyAsync<T>(HttpContext ht
                 }
                 try
                 {
-                    var bodyValue = await httpContext.Request.ReadFromJsonAsync<T>();
-                    if (!allowEmpty && bodyValue == null)
-                    {
-                        if (!isInferred)
-                        {
-                            logOrThrowExceptionHelper.RequiredParameterNotProvided(parameterTypeName, parameterName, "body");
-                        }
-                        else
-                        {
-                            logOrThrowExceptionHelper.ImplicitBodyNotProvided(parameterName);
-                        }
-                        httpContext.Response.StatusCode = StatusCodes.Status400BadRequest;
-                        return (false, bodyValue);
-                    }
-                    return (true, bodyValue);
+                    bodyValue = await httpContext.Request.ReadFromJsonAsync<T>();
```

Comment:
https://github.com/dotnet/aspnetcore/pull/48269#pullrequestreview-1438729332

Tracked anywhere?

Reply:
I've opened https://github.com/dotnet/aspnetcore/pull/48573 to follow-up on this.